hw/top_earlgrey/data/ip/chip_entropy_src_testplan.hjson

```diff
@@ -0,0 +1,195 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_entropy_src
+  testpoints: [
+    // ENTROPY_SRC (pre-verified IP) integration tests:
+    {
+      name: chip_sw_entropy_src_ast_rng_req
+      desc: '''Verify the RNG req to AST.
+
+            - Enable ENTROPY_SRC in FIPS mode.
+            - Enable Firmware Override - Extract & Insert to route entropy received from AST to the observe FIFO.
+            - Verify that the observe FIFO depth is non-zero via SW - indicating the reception of data over the AST RNG interface.
+            - Verify the correctness of the received data with assertion based connectivity checks.
```

Comment:
We can't check whether the values match in silicon. But checking that the Observe FIFO fills up is a very fundamental test prior to validation testing. Since the test is already there I've marked this as SV1. This is assuming Nuvoton will use this for validating the noise source.

Reply:
I think it's worth mentioning this in the *Notes for silicon targets* below

Reply:
Good idea. I've added this information there as well.